### PR TITLE
Genius2rocky

### PR DESCRIPTION
**Pull request by jooghe**
Can you check the additional info regarding cluster modules and merge?

### CODE REVIEW

File: source/leuven/genius_2_rocky.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Genius migration to Rocky 8
+===========================
+The :ref:`Genius <genius hardware>` operating system will be changed from CentOS 7 to Rocky 8. This is the same OS as currently running on :ref:`wICE <wice hardware>`.
+The migration will be done on September 25.
+Sofware installed on wICE is not impacted, because there is no change of OS on wICE.
+
+.. note::
+
+   Tests nodes are available. If you want to test your software or your Conda environment you can do so as follows:
```
Comment (mselwa) [inline source/leuven/genius_2_rocky.rst:11]: Test nodes

File: source/leuven/genius_2_rocky.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+This can be done on one of the available test nodes. Please request access to the lpt2_rocky8_pilot group.
+
+.. _impact_on_conda:
+
+Impact on Conda environments
+----------------------------
+The Conda environment you installed might need reinstallations. If you already have a Conda environment that works on wICE, it also should work on Genius after the migration.
+If you only have a Conda environment working on Genius, it's best to create a new Conda installation. In this case, it is recommended to recreate your environment for full compatibility with the new OS. Best practice is to choose a new installation folder with explicit mention of the new OS, e.g. ::
+
+   ${VSC_DATA}/miniconda3-rocky
```
Comment (mselwa) [inline source/leuven/genius_2_rocky.rst:60]: I would rewrite the lines that a lot of people asked me from the docs - how to change the path in miniconda install:

bash Miniconda3-latest-Linux-x86_64.sh -b -p $VSC_DATA/miniconda3/rocky
export PATH="${VSC_DATA}/miniconda3-rocky/bin:${PATH}"

File: source/leuven/genius_2_rocky.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  
+You can then use this Conda environment after the migration. You can prepare this on the test nodes.
+
+
+.. _impact_on_starting_jobs:
+
+Impact on starting jobs on Genius and wICE
+------------------------------------------
+In order to minimize the changes you need to make to your jobscripts, an appropriate module path (``$MODULEPATH``) will be set by default at the start of your job of the the migration of Genius to Rocky 8 OS. This new module path will now contain all toolchain versions, starting from 2018a on Genius and starting from 2021a on wICE. This is an important change! Previously the modulepath was set to a single toolchain, that would not change over time. You might have set a module path in your jobscripts to refer to newer toolchains.
```
Comment (mselwa) [inline source/leuven/genius_2_rocky.rst:69]: maybe add excplicitely
... all toolchain versions, so all the modules, starting from...

Module use and module unuse commands will not be necessary to see the most recent versions of the mdules.

File: source/leuven/genius_2_rocky.rst [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+will set the module path for the modules that are applicable for the Genius ``batch`` partition::
+
+   $ module load cluster/wice/batch
+
+will set the the module path of the wICE ``batch`` partition. When you do this on the login node you can examine wich modules are available with the regular commands, e.g.::
+
+   $ module avail
+   $ module spider cp2k
+
```
Comment (mselwa) [inline source/leuven/genius_2_rocky.rst:116]: It does not mean that these modules are available and could be used on login nodes.

**Review by mselwa — APPROVED** [review]
Some comments that you can include or not - as you decide.

File: source/leuven/genius_2_rocky.rst (hunk 4 above)
Comment (jooghe) [reply source/leuven/genius_2_rocky.rst:116]: I hope that was clear from the "(only for experimentation, not for actual computing)". Rest of the coments I've added.

**Review by WouterVanAssche — COMMENTED** [review]
Seems fine to me.

**Pull request merged**